src/pyhf/infer/test_statistics.py

```diff
@@ -2,6 +2,48 @@
 from .mle import fixed_poi_fit, fit
 from ..exceptions import UnspecifiedPOI
 
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def _qmu_like(mu, data, pdf, init_pars, par_bounds):
+    """
+    Clipped version of _tmu where _qmu = 0 if muhat > 0 else _tmu
+
+    If the lower bound of the POI is 0 this automatically implments
+    qmu_tilde. Otherwise this is qmu (no tilde).
+    """
+    tensorlib, optimizer = get_backend()
+    tmu_stat, (mubhathat, muhatbhat) = _tmu_like(
```

Comment:
`mubhathat` is not used at all

```suggestion
    tmu_stat, (_, muhatbhat) = _tmu_like(
```